prep-css-box-model/styles.css

```diff
@@ -0,0 +1,18 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+  margin: 20px;
+  padding: 10px;
+
+  p.double {
+    border-style: double;
+    width: 3px;
+    color: darkgray;
+  }
```

Comment:
The example in the documentation is for paragraphs, but we just want to add properties to our rule set for the button

Reply:
Just updated the correct ode (I think)